.circleci/src/workflows/native_production_delivery.yml

```diff
@@ -248,12 +203,3 @@ jobs:
         - deliver_integreat_ios
         - deliver_malte_ios
         - deliver_aschaffenburg_ios
-
-  - move_release_notes:
```

Comment:
Are you sure you can delete move_release_notes here and everything just works fine? If triggering a native_production_delivery, the release_notes are not moved to the correct subfolder, or am I missing something here?

Just a reminder: ios does not promote the app from testflight to production and just reuses the release-notes from the beta as android does.

Reply:
So if using this workflow, in native the wrong release notes are displayed?

Reply:
No release notes are displayed, not the wrong release notes.